src/d2_docker/config/dhis2-core-start.sh

```diff
@@ -34,6 +35,8 @@ debug() {
 
 run_sql_files() {
     base_db_path=$(test "${LOAD_FROM_DATA}" = "yes" && echo "$root_db_path" || echo "$post_db_path")
+    debug "Files in data path"
+    find "$base_db_path" >&2;
```

Comment:
[minor] Debug only?

Reply:
I think it may be always useful, I wanted to add this some time ago. It's pretty common that the DB does not load as you expected (the dump extension file is not right, whatever), this way we can see exactly what's in the data folder.